### DISCUSSION TITLE
macOS: Inlay hints

### DISCUSSION THREAD

**Discussion by CryptoCrocodile** [microsoft/vscode-cpptools#10143] [General]
Are inlay hints support on the macos/clang configuration? I can't seem to get them to work.

Thanks

**Comment by CryptoCrocodile** [comment]
Ah, I believe this might be related to having files in a symlink'd folder

**Comment by michelleangela** [comment]
@thisisagenericusernameforagenericperson 
Could you ensure `editor.inlayHints.enabled` is `on` ? There are also specific inlay hints settings `C_Cpp.inlayHints.*` for the C++ extension.

If those inlay settings are enabled, verify IntelliSense is working for the source file's translation unit (TU) by checking if other IntelliSense features, such as auto complete, is working.

**Comment by H-G-Hristov** [comment]
@thisisagenericusernameforagenericperson Switch to the current preview version or wait for it to be released.

**Reply by sean-mcmanus** [reply]
@H-G-Hristov I don't understand your comment. Are you referring to something that is fixed with 1.13.3?

**Reply by H-G-Hristov** [reply]
I guess it depends on the users' project configuration, e.g. they are using universal binaries. You implemented a bunch of fixes in the 1.13.x series which improved my experience on macOS significantly. Thank you for your work!
